common/etl/collector.go

```diff
@@ -122,7 +123,7 @@ func loadFilesIntoBucket(db ethdb.Database, bucket []byte, providers []dataProvi
 				return err
 			}
 			runtime.ReadMemStats(&m)
-			log.Info(
+			log.Debug(
```

Comment:
These are actually quite useful to see the progress

Reply:
What I am really saying - can we keep it as `Info`? It really helps me when I am running these long processes. And if I turn on debug, it adds quite a bit more stuff into the logs

Reply:
Sure. Don’t know why it’s still debug. Will change.